Content.Server/Dragon/DragonRiftSystem.cs

```diff
@@ -33,11 +34,20 @@ public override void Initialize()
     {
         base.Initialize();
 
+        SubscribeLocalEvent<DragonRiftComponent, ComponentGetState>(OnGetState);
         SubscribeLocalEvent<DragonRiftComponent, ExaminedEvent>(OnExamined);
         SubscribeLocalEvent<DragonRiftComponent, AnchorStateChangedEvent>(OnAnchorChange);
         SubscribeLocalEvent<DragonRiftComponent, ComponentShutdown>(OnShutdown);
     }
 
+    private void OnGetState(EntityUid uid, DragonRiftComponent component, ref ComponentGetState args)
+    {
+        args.State = new DragonRiftComponentState
+        {
+            State = component.State,
```

Comment:
Use `Entity<T>` for event handlers.
```suggestion
    private void OnGetState(Entity<DragonRiftComponent> ent, ref ComponentGetState args)
    {
        args.State = new DragonRiftComponentState
        {
            State = ent.Comp.State,
```